doc/src/BCMPackage.rst

```diff
@@ -10,6 +10,11 @@ bcm_find_package
 
 This works the same as cmake's ``find_package`` except in addition, it will keep track of the each call to ``bcm_find_package`` in the project. Functions like ``bcm_package`` and ``bcm_boost_package`` will include these dependencies automatically in cmake's package config that gets generated.
 
+.. I don't understand why 'bcm_find_package' needs to do anything other than
+   what the standard find_package does. Can't the targets exported by
+   'find_package' carry all the "package config" information needed for '.pg'
+   file generation?
+
```

Comment:
This is to track the dependency, it works exactly as `find_package` does except it stores this in a variable, so when 'foo-config.cmake' is generated, it will generate a call `find_dependency` for each `bcm_find_package` that is called. This only happens with `bcm_boost_package` or `bcm_package`. When using `bcm_auto_export` the user will still need to explicitly define each dependency.